### DISCUSSION TITLE
Laragon link auto redirect {name}.test/dashboard

### DISCUSSION THREAD

**Discussion by mol3x** [leokhoa/laragon#746] [General]
![Screenshot 2024-04-02 184540](https://github.com/leokhoa/laragon/assets/72003120/a08a5257-323c-4b32-813c-4fc48d2a2261)
Why can't my project be opened, the project auto redirects like in the picture

**Comment by tsukiforge** [comment]
less, that's what cakrawala.test/dashboard/home should be
